LambdaEngine/Source/Rendering/RenderGraph.cpp

```diff
@@ -2132,6 +2139,10 @@ namespace LambdaEngine
 		m_PipelineStageCount = (uint32)pipelineStageDescriptions.GetSize();
 		m_pPipelineStages = DBG_NEW PipelineStage[m_PipelineStageCount];
 
+		Profiler::GetGPUProfiler()->Init(GPUProfiler::TimeUnit::MICRO);
+		Profiler::GetGPUProfiler()->CreateTimestamps(m_PipelineStageCount * m_BackBufferCount);
```

Comment:
This should probably be times m_PipelineStageCount * m_BackBufferCount * 2, since we have Compute and Graphics that we would like to create timestamps for

Reply:
Fixed on local